project/src/main/webapp/scripts/listAssignmentSubmissions.js

```diff
@@ -0,0 +1,12 @@
+async function listAssignmentSubmissions() {
+
+    let servletURL = "/listAssignmentSubmissions";
+
+    let response = await fetch(servletURL, {
+        method: "GET",
+        mode: "no-cors"
+    });
+
+    console.log("\n\nBING\tBANG\tBONG\n\n");
```

Comment:
Debugging logs should not be checked in

Reply:
My bad, I thought I'd removed them all, I'll make sure to check for them before committing.